# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/RenderTextureHost.h
# Commit: 49ae9de85ab9
# Full Hash: 49ae9de85ab90bf13b9c6bc2c8b6bb540140c90d
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2020-11-06 04:14:34
# Regressor Bug: 1673983
# File Overlap Count: 2
# Description:
#   Bug 1673983 - Support using D3D11 textures as external compositor surfaces with D3D11 compositor. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D95800
# ==============================================================================

diff -r 91d8f0f829fd -r 49ae9de85ab9 gfx/webrender_bindings/RenderTextureHost.h
--- a/gfx/webrender_bindings/RenderTextureHost.h	Thu Nov 05 20:18:13 2020 +0000
+++ b/gfx/webrender_bindings/RenderTextureHost.h	Thu Nov 05 19:27:23 2020 +0000
@@ -25,6 +25,7 @@
 
 class RenderCompositor;
 class RenderDXGITextureHost;
+class RenderDXGIYCbCrTextureHost;
 class RenderMacIOSurfaceTextureHost;
 class RenderBufferTextureHost;
 class RenderTextureHostSWGL;
@@ -69,6 +70,9 @@
   virtual bool SyncObjectNeeded() { return false; }
 
   virtual RenderDXGITextureHost* AsRenderDXGITextureHost() { return nullptr; }
+  virtual RenderDXGIYCbCrTextureHost* AsRenderDXGIYCbCrTextureHost() {
+    return nullptr;
+  }
 
   virtual RenderMacIOSurfaceTextureHost* AsRenderMacIOSurfaceTextureHost() {
     return nullptr;